### DISCUSSION TITLE
Will the future community toolkit provide a new control template style for the SelectorBar control in Winui 3 1.5?

### DISCUSSION THREAD

**Discussion by Gaoyifei1011** [CommunityToolkit/Windows#381] [Q&A]
### Describe the bug

Will the future community toolkit provide a new control template style for the SelectorBar control in Winui 3 1.5?
未来社区工具包是否会给 Winui 3 1.5 的 SelectorBar 控件提供新的控件模板样式？

Currently, in Winui 3 1.5, the WinUI team has added a new SelectorBar control that looks and functions similar to the PivotSegmentedStyle in the Segmented control in the community toolkit. Will there be other styles for the SelectorBar Segmented control in a future release?
目前，在 Winui 3 1.5 时，WinUI 团队新添加了一个 SelectorBar 控件，这个控件的外观和功能类似于社区工具包Segmented 控件的 PivotSegmentedStyle，未来版本中是否会为 SelectorBar 控件提供 Segmented 控件其他的样式呢？

https://learn.microsoft.com/zh-cn/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.selectorbar?view=windows-app-sdk-1.5

### Steps to reproduce

```text
None
```

### Expected behavior

See describe

### Screenshots

![image](https://github.com/CommunityToolkit/Windows/assets/49179966/a5d5e414-31e1-403d-8681-af9f21a55912)

### Code Platform

- [ ] UWP
- [X] WinAppSDK / WinUI 3
- [ ] Web Assembly (WASM)
- [ ] Android
- [ ] iOS
- [ ] MacOS
- [ ] Linux / GTK

### Windows Build Number

- [ ] Windows 10 1809 (Build 17763)
- [ ] Windows 10 1903 (Build 18362)
- [ ] Windows 10 1909 (Build 18363)
- [ ] Windows 10 2004 (Build 19041)
- [ ] Windows 10 20H2 (Build 19042)
- [ ] Windows 10 21H1 (Build 19043)
- [ ] Windows 10 21H2 (Build 19044)
- [ ] Windows 10 22H2 (Build 19045)
- [ ] Windows 11 21H2 (Build 22000)
- [X] Other (specify)

### Other Windows Build number

22621

### App minimum and target SDK version

- [ ] Windows 10, version 1809 (Build 17763)
- [ ] Windows 10, version 1903 (Build 18362)
- [ ] Windows 10, version 1909 (Build 18363)
- [ ] Windows 10, version 2004 (Build 19041)
- [ ] Windows 10, version 2104 (Build 20348)
- [ ] Windows 11, version 22H2 (Build 22000)
- [X] Other (specify)

### Other SDK version

22621

### Visual Studio Version

2022

### Visual Studio Build Number

17.9

### Device form factor

Desktop

### Additional context

None

### Help us help you

Yes, I'd like to be assigned to work on this item.

**Comment by insomniachi** [comment]
Also SelectorBar does not support binding itemsource. i had to write my own behavior

**Comment by Gaoyifei1011** [comment]
> Also SelectorBar does not support binding itemsource. i had to write my own behavior

The wrapper it uses internally is not listviewbase but itemsview (winui3 specific)

**Comment by Poker-sang** [comment]
The `SelectorBar` and `Segmented` functions are pretty much duplicated. I would also like to be able to reduce the duplication by adding the `Style` of the `SelectorBar`
